code/modules/clothing/glasses/engine_goggles.dm

```diff
@@ -154,8 +162,59 @@
 
 	modes = list(MODE_NONE = MODE_SHUTTLE, MODE_SHUTTLE = MODE_NONE)
 
+
+/obj/item/clothing/glasses/meson/engine/atmos_imaging
+	name = "Atmospheric thermal imaging goggles"
+	desc = "Goggles used by Atmospheric Technician to see the thermal energy of gasses in open areas"
+	icon_state = "trayson-"
```

Comment:
this is actually your icon state?
name also shouldn't start capitalized
also do we need these glasses at all now? they were added to t-ray

Reply:
I would like for there to be a seperate version, cycling through the modes can be a hassle

Reply:
please fix the name and icon state then